feelpp/cmake/modules/feelpp.module.petsc.cmake

```diff
@@ -82,7 +85,7 @@ if ( PETSC_HAVE_MUMPS )
   set(FEELPP_HAS_MUMPS 1)
   set(FEELPP_PETSC_ENABLED_OPTIONS "${FEELPP_PETSC_ENABLED_OPTIONS} MUMPS")
 else()
-  message( FATAL_ERROR "PETSc should be compiled with MUMPS")
+  message( STATUS "PETSc should be compiled with MUMPS")
```

Comment:
PETSc should be compiled with Mumps

Reply:
unfortunately fedora petsc version is not compiled with mumps support

Reply:
maybe  replace with a warning ?

Reply:
I think no, an error will be preferred else we didn't ensure that all work correctly, this is the reason we have added this.
You can put a cmake variable (or fedora detection in cmake) if you want to use a petsc without mumps

Reply:
see #1981